README.md

```diff
@@ -37,7 +37,9 @@ If you still cannot get the tool to work, join our [Discord Server](https://disc
 
 ### Running The Ship of Harkinian
 
-Launch the game. If the window immediately closes, or if there are visual artifacts, you may have selected the wrong rom in the OTRGui tool. 
+Launch the game. If the window immediately closes, or if there are visual artifacts, you may have selected the wrong rom in the OTRGui tool.
+
+You can enable a taskbar by pressing F1, and you can enter fullscreen mode by pressing F11 (on OpenGL) or Alt+Enter (on DirectX).
```

Comment:
I think it's F10, not F11.  Can't test right this second though.

Reply:
Some people have said that F10 works, other people say F11 works

Reply:
it's f10 not f11